pkg/logutil/logutil.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import "log"
+
+// Objects implementing the Logger interface can be configured to log messages
+// in an output format based on a set of flags.
+type Logger interface {
+	SetFlags(flag int)
+	Flags() int
+}
+
+// Config holds the logger config.
+type Config struct {
+	DisableTimestamps bool
+}
+
+// InitLogger initializes a Logger based on the specified config.
+func InitLogger(logger Logger, config *Config) {
+	flags := getFlags(config)
```

Comment:
same here, we can only deal with nil case (golang.Log) for now. Add a todo would be good enough.